### DISCUSSION TITLE
Please update the docker image.

### DISCUSSION THREAD

**Discussion by gerasghulyan** [willnorris/imageproxy#323] [General]
The docker image has not been updated after the latest changes.
Can you please update the docker image as well?

Thanks
Geras

**Comment by willnorris** [comment]
Which image are you looking at?  A month or two ago I switched from publishing on [Docker Hub](https://hub.docker.com/r/willnorris/imageproxy) to [GitHub Container Registry](https://github.com/willnorris/imageproxy/pkgs/container/imageproxy) instead.  The [`latest`](https://github.com/willnorris/imageproxy/pkgs/container/imageproxy/8805416?tag=latest) tag has the most recent tagged released (which is a month or so old right now), and the [`main`](https://github.com/willnorris/imageproxy/pkgs/container/imageproxy/9888520?tag=main) tag has all commits on the main branch, including your change from last week.

**Reply by gerasghulyan** [reply]
Thank you,
I looked at the old place.

Regards
Geras

**Comment by desimpson** [comment]
Hi @willnorris,

I was hoping you could please provide a new image build? There's a few CVEs in [the most recent image](https://github.com/willnorris/imageproxy/pkgs/container/imageproxy/223956223?tag=sha256-e0708b2c231f8e91a898d4100ee23279907e89d8b8b27d12b6d39ae5f1640bd7.sig):

```
12 vulnerabilities found in 4 packages
  UNSPECIFIED  1  
  LOW          2  
  MEDIUM       3  
  HIGH         5  
  CRITICAL     1  
```

Thanks.
